docs/developer-guide/syntax-reference.rst

```diff
@@ -939,6 +939,35 @@ example of converting a BIGINT into a VARCHAR type:
       WINDOW TUMBLING (SIZE 20 SECONDS)
       GROUP BY page_id;
 
+CASE
+~~~~
+
+ **Synopsis**
+
+ .. code:: sql
+
+     CASE
+        WHEN condition THEN result
+        [ WHEN ... ]
```

Comment:
Of lower priority, but it’d be nice if we could also make it more clear that any number of `WHEN … THEN …` clauses are allowed. Perhaps either
```
WHEN condition THEN result
[ WHEN … THEN … ]
…
[ WHEN … THEN … ]
[ ELSE result ]
```
or updating the example to use more than two when clauses?

Reply:
Added your suggestion!